velox/expression/tests/utils/ArgumentTypeFuzzer.cpp

```diff
@@ -21,20 +21,82 @@
 
 #include "velox/expression/ReverseSignatureBinder.h"
 #include "velox/expression/SignatureBinder.h"
+#include "velox/expression/type_calculation/TypeCalculation.h"
+#include "velox/type/SimpleFunctionApi.h"
 #include "velox/type/Type.h"
 #include "velox/vector/fuzzer/VectorFuzzer.h"
 
 namespace facebook::velox::test {
 
 std::string typeToBaseName(const TypePtr& type) {
-  return boost::algorithm::to_lower_copy(std::string{type->kindName()});
+  return type->isDecimal()
```

Comment:
It corresponds to the baseName from TypeSignature, which comes from parseTypeSignature.
https://github.com/facebookincubator/velox/blob/acb57999cec11f8a5741a2c8d2536ff6781c4c02/velox/expression/tests/ExpressionFuzzer.cpp#L682
To align with the result of parseTypeSignature, decimal instead of bigint or hugeint is returned here. DATE works as parseTypeSignature returns `integer`, so to use kindName is consistent.
[Link to ParseTypeSignatureTest](https://github.com/facebookincubator/velox/blob/main/velox/expression/signature_parser/tests/ParseTypeSignatureTest.cpp#L92-L117)